integrationtests/python_src/px4_it/mavros/mission_test.py

```diff
@@ -48,18 +48,51 @@
 import sys
 import os
 import time
+import glob
+import json
 
 import mavros
 from pymavlink import mavutil
 from mavros import mavlink
 
+import px4tools
+
 from geometry_msgs.msg import PoseStamped
 from mavros_msgs.srv import CommandLong, WaypointPush
 from mavros_msgs.msg import Mavlink, Waypoint, ExtendedState
 from sensor_msgs.msg import NavSatFix
 from mavros.mission import QGroundControlWP
 #from px4_test_helper import PX4TestHelper
 
+def get_last_log():
```

Comment:
This function is how I get the last log file created. There is probably a better more robust way.